test/Microsoft.SqlTools.ServiceLayer.UnitTests/Utility/TestObjects.cs

```diff
@@ -160,7 +160,36 @@ public override int ExecuteNonQuery()
 
         public override object ExecuteScalar()
         {
-            throw new NotImplementedException();
+            // Tailored for Row Delete tests
```

Comment:
I don't think this comment is necessary. It's not important that edit data is the only thing using this currently and should have no impact on the implementation here. 

I would suggest adding a comment explaining what this code is doing since it's not directly clear what value this would be returning.